.teamcity/src/main/kotlin/common/VersionedSettingsBranch.kt

```diff
@@ -2,19 +2,32 @@ package common
 
 import jetbrains.buildServer.configs.kotlin.v2019_2.DslContext
 
-data class VersionedSettingsBranch(val branchName: String) {
+data class VersionedSettingsBranch(val branchName: String, val enableTriggers: Boolean) {
 
     companion object {
-        val MASTER = VersionedSettingsBranch("master")
-        val EXPERIMENTAL = VersionedSettingsBranch("experimental")
+        private
+        const val MASTER_BRANCH = "master"
+        private
+        const val RELEASE_BRANCH = "release"
+        private
+        const val EXPERIMENTAL_BRANCH = "experimental"
+        private
+        val mainBranches = setOf(MASTER_BRANCH, RELEASE_BRANCH)
 
         fun fromDslContext(): VersionedSettingsBranch {
             val branch = DslContext.getParameter("Branch")
             // TeamCity uses a dummy name when first running the DSL
             if (branch.contains("placeholder-1")) {
-                return MASTER
+                return VersionedSettingsBranch(MASTER_BRANCH, true)
             }
-            return VersionedSettingsBranch(branch.toLowerCase())
+            return VersionedSettingsBranch(branch.toLowerCase(), mainBranches.contains(branch.toLowerCase()))
```

Comment:
Does this mean `release6x` is no longer triggering new builds when pushed by developers?